supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -246,7 +247,7 @@ There is no functional difference between the first server that was installed an
 [discrete]
 [[iiop-openjdk]]
 ==== image:images/yes.png[yes] iiop-openjdk subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "iiop-openjdk" subsystem is used to configure Common Object Request Broker Architecture (CORBA) services. In general text, write in lowercase as two words separated by a hyphen. Use "IIOP subsystem" when referring to the iiop-openjdk subsystem in titles and headings.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _iiop-openjdk_ subsystem is used to configure Common Object Request Broker Architecture (CORBA) services. In general text, write in lowercase as two words separated by a hyphen. Use "IIOP subsystem" when referring to the `iiop-openjdk` subsystem in titles and headings.
```

Comment:
In other entries for EAP subsystems, the entire thing is italicized. So should be _iiop-openjdk subsystem_ in the first sentence

Reply:
fixed